tests/qos/test_pfc_pause.py

```diff
@@ -231,7 +232,11 @@ def test_pfc_pause_lossless(pfc_test_setup, fanouthosts, duthost, ptfhost,
     """ DSCP values for other lossless priority """
     other_lossless_dscps = lossless_prio_dscp_map[other_lossless_prio]
     """ DSCP values for lossy priorities """
-    lossy_dscps = list(set(range(64)) - set(other_lossless_dscps) - set(dscp))
+    if 'backend' in setup['testbed_type']:
+        prio_range = 8
+    else:
+        prio_range = 64
```

Comment:
suggest put prio_range calculation in a helper function as the same if - else block used twice in this file.

Reply:
Done!